documentation/modules/proc-configuring-kafka-user.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// assembly-using-the-user-operator.adoc
+
+[id='proc-configuring-kafka-user-{context}']
+= Configuring a Kafka user
+
+Use the properties of the `KafkaUser` resource to configure a Kafka user.
+
+You can use `kubectl apply` to create or modify users, and `kubectl delete` to delete existing users.
+
+For example:
+
+* `kubectl apply -f _<user-config-file>_`
+* `kubectl delete KafkaUser _<user-name>_`
+
+When you configure the `KafkaUser` authentication and authorization mechanisms, ensure they match the equivalent `Kafka` configuration:
+
+* `KafkaUser.spec.authentication` matches `Kafka.spec.kafka.listeners.*.authentication`
+* `KafkaUser.spec.authorization` matches `Kafka.spec.kafka.authorization`
+
+
+a `KafkaUser`, you use the `spec` property to define the `authentication` and `authorization` mechanisms required for the Kafka user to be able to connect to Kafka brokers.
```

Comment:
**As** a `KafkaUser`...

Reply:
Actually, that line was meant to be taken out pending changes in previous paragraph. Removed now.